File: python/get_whitelisted_ips.py
#!/usr/bin/env python3

from _client import get
import os

# Get whitelisted IPs for a trunk
trunk_sid = os.getenv('TRUNK_SID')
if not trunk_sid:
    print("Error: TRUNK_SID environment variable is required")
    exit(1)

print(f"Getting whitelisted IPs for trunk {trunk_sid}...")
result = get(f'/trunks/{trunk_sid}/whitelisted-ips')
print("Whitelisted IPs retrieved successfully!") 

File: php/get_whitelisted_ips.php
#!/usr/bin/env php
<?php

require_once '_client.php';

// Get whitelisted IPs for a trunk
$trunk_sid = getenv('TRUNK_SID');
if (!$trunk_sid) {
    echo "Error: TRUNK_SID environment variable is required\n";
    exit(1);
}

echo "Getting whitelisted IPs for trunk $trunk_sid...\n";
$result = exo_get("/trunks/$trunk_sid/whitelisted-ips");
echo "Whitelisted IPs retrieved successfully!\n";
?> 